apps/hash-graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -658,9 +664,9 @@ where
             StatusCode::INTERNAL_SERVER_ERROR
         })?;
 
-    let subject = match owner_id {
-        OwnerId::Account(id) => EntityDirectOwnerSubject::Account { id },
-        OwnerId::AccountGroupMembers(id) => EntityDirectOwnerSubject::AccountGroup {
+    let owner = match owner_id {
+        WebSubject::Account(id) => EntityDirectOwnerSubject::Account { id },
```

Comment:
why not `WebSubjectId`? and `xId` generally when we are talking about ids and not the whole thing

Reply:
I did not add `Id` to these terms as the ID _is_ the subject in the permission implementation. For `AccountGroup` this is `id` and `set` so this is no longer an ID only.